storage/google/src/frontend/Helpers.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import axios from "axios";
+
+import {
+  FrontendTransferData,
+  FrontendTransferType,
+  FrontendUrlDownloadInput,
+} from "@itwin/object-storage-core/lib/frontend";
+
+export const uploadFileSizeLimit = 5_000_000_000; // 5GB
```

Comment:
Is this limit set somewhere?

Reply:
It's not used. I've removed it and this file and merged the changes into original frontend Helpers file.